app/services/katello/registration_manager.rb

```diff
@@ -264,12 +264,20 @@ def set_host_collections(host, activation_keys)
         host.host_collection_ids = host_collection_ids
       end
 
-      def lookup_content_view_environment(activation_keys)
+      def lookup_content_view_environments(activation_keys)
+        if Setting['allow_multiple_content_views']
+          cves = activation_keys.reverse.map do |act_key|
```

Comment:
As discussed let's remove the `.reverse` here. Users wouldn't expect reverse order.

Reply:
```suggestion
          cves = activation_keys.map do |act_key|
```